src/main/java/com/sgoertzen/sonarbreak/SonarBreakMojo.java

```diff
@@ -37,13 +40,19 @@ protected static String buildErrorString(List<Condition> conditions) {
         return builder.toString();
     }
 
+    @Override
     public void execute() throws MojoExecutionException {
         MavenProject mavenProject = getMavenProject();
         String version = mavenProject.getVersion();
-        if (sonarKey == null || sonarKey.equals("")) {
+        if (StringUtils.isEmpty(sonarKey)) {
 
             sonarKey = String.format("%s:%s", mavenProject.getGroupId(), mavenProject.getArtifactId());
         }
+        if (!StringUtils.isEmpty(sonarBranch)){
+
+            sonarKey = String.format("%s:%s", sonarKey, sonarBranch);
+        }
+
```

Comment:
Thanks for doing this!  Can you add an integration test for this new functionality into the integration tests folder?  I depend on those when I test this against new versions of Sonar so I don't have to do any manual testing.  You can see an example of one I did for the sonarKey here: https://github.com/sgoertzen/sonar-break-maven-plugin/tree/master/integration-tests/custom-key